### PR TITLE
RHEL-07-020260

### PR DESCRIPTION
**Pull request by robinje**
The string 'rhel-7-server-optional-rpms' is hard-coded in the underlying infrastructure which is causing a failure in the RHEL-07-020260. 

'find . -type f xargs grep rhel-7-server-optional-rpms' only reviled the results file.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -119,7 +119,7 @@ rhel_07_020100: true
 rhel_07_020101: true
 rhel_07_020110: true
 rhel_07_020240: true
-rhel_07_020260: true
+rhel_07_020260: false
```
Comment (aheifetz) [inline defaults/main.yml:122]: we just turned off the test - is there a way to fix it ?

Comment (robinje) [reply defaults/main.yml:122]: Go through the YUM source code, find the literal string and replace it with the the desired string, recompile it and use the new version.

**Review by PaulWaldo — APPROVED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (Dav1dV) [reply defaults/main.yml:122]: @aheifetz I'm assuming this is now not a blocker to approval & approving

**Review by Dav1dV — APPROVED** [review]
Since conversation appears to be resolved 4 now for this PR

**Pull request merged**